config/hathitrust_config.rb

```diff
@@ -77,6 +77,10 @@ def superset_volume_writer
         )
       end
 
+      def profiles
```

Comment:
this is now duplicated in private and public. It was formerly private, but the datasets_cli needs to know about it unless it's going to special case the subsets vs. superset which I'd rather try to avoid unless necessary